src/runner/index.js

```diff
@@ -735,6 +748,12 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
+    skipJsErrors (opts, deps) {
+        this._options[OPTION_NAMES.skipJsErrors] = ensureSkipJsErrorsCallbackWrapped(opts, deps);
```

Comment:
We discussed  this method and decided to remove it to leave only one way to assign skipJsErrors options in runner to avoid redundant pluralism. Because otherwise it may confuse users and requires more effort while writing documentation.